src/javascript/app_2/Modules/Trading/Components/Form/screen_large.jsx

```diff
@@ -5,27 +5,26 @@ import Purchase     from '../../Containers/purchase.jsx';
 import TradeParams  from '../../Containers/trade_params.jsx';
 import UILoader     from '../../../../App/Components/Elements/ui_loader.jsx';
 
-const ScreenLarge = ({ is_trade_enabled }) => (
-    <div className='sidebar-container desktop-only'>
-        <div className='sidebar-items'>
-            {!is_trade_enabled ?
-                <UILoader />
-                :
-                <React.Fragment>
-                    <fieldset className='trade-types'>
-                        <ContractType />
-                    </fieldset>
-                    <TradeParams />
-                    <div className='purchase-wrapper'>
-                        <Purchase />
-                    </div>
-                </React.Fragment>
-            }
-        </div>
+const ScreenLarge = ({ contract_id, is_trade_enabled }) => (
```

Comment:
Same here, since [`is_contract_visible `](https://github.com/binary-com/binary-static/pull/4198/files#diff-3b34b91aeda0995b1b6fb911a136d973R14) is the property name passed from `form_layout.jsx`. Probably it's not needed anymore?